config/routes.rb

```diff
@@ -12,6 +12,10 @@
         get :status, to: 'api#status'
         post 'users/facebook', to: 'sessions#facebook'
         resources :matches, only: :index
+        resources :users, only: :show
+        resource :user, only: :update do
```

Comment:
This is not RESTful, please remove it. Also take a look to [this article](https://thoughtbot.com/upcase/videos/rest) about REST.